fineract-provider/src/main/java/org/apache/fineract/infrastructure/jobs/filter/LoanCOBApiFilter.java

```diff
@@ -66,6 +66,7 @@ public class LoanCOBApiFilter extends OncePerRequestFilter implements BatchFilte
 
     private static final List<HttpMethod> HTTP_METHODS = List.of(HttpMethod.POST, HttpMethod.PUT, HttpMethod.DELETE);
 
+    public static final Pattern IGNORE_LOAN_PATH_PATTERN = Pattern.compile("\\/?loans\\/catch-up");
```

Comment:
question mark, because if it's exists it will match to "/any_text-before_loans/catch-up" URL, not just the "/loans/catch-up"